requirements/docs.in

```diff
@@ -12,4 +12,4 @@ sphinx
 sphinx-rtd-theme
 
 # Needed to build markdown docs
-recommonmark==0.5.*  # https://github.com/zulip/zulip/issues/11395
+recommonmark==0.6.*  # https://github.com/zulip/zulip/issues/11395
```

Comment:
The comment is now obsolete. We should replace this line with just

```
recommonmark
```

as we’ll now want to future versions of `recommonmark` to be considered for upgrades when we do them, unless we find a different reason not to. (The `.txt` files will still record that our current version is 0.6.0.)